guides/index.md

```diff
@@ -20,10 +20,11 @@ The CLI comes with a command-line-based help system too. At any point, if you're
 
 There are thousands of JavaScript libraries that work great in Ember. When an [npm package](https://www.npmjs.com/) offers some Ember-specific conveniences, we call it an “addon.” Ember CLI’s addon system provides a way to create reusable units of code, share components and styling, extend the build tooling, and more — all with minimal configuration.
 
-To view some of the most popular addons, visit [Ember Observer](https://emberobserver.com). 
+To view some of the most popular addons, visit [Ember Observer](https://emberobserver.com).
 
-You can still use your favorite npm packages directly too. If they are not available as addons yet, you can use
-a community tool called [ember-auto-import](https://github.com/ef4/ember-auto-import), include them in the build pipeline yourself, or create a wrapper addon of your own.
+Although Ember-specific packages are more specific, you can use your favorite npm packages directly too.
+By default, Ember apps created through Ember CLI include the `ember-auto-import` dependency, which enables importing npm packages directly. For example, if you want to use the Highcharts library in your application,
```

Comment:
```suggestion
By default, Ember apps created through Ember CLI include the `ember-auto-import` dependency, which enables importing npm packages directly. For example, if you want to use the `highcharts` library in your application,
```

Words like this can be either added to the dictionary or wrapped in backticks. Things in backticks are ignored by our spellchecker in the test suite. Since this is a library and it's only mentioned once, I think it would be fine to wrap it in backticks, and then the tests will pass.